CMakeLists.txt

```diff
@@ -140,7 +152,7 @@ endif()
 
 if (NOT BUILD_SHARED_LIBS)
     if (NOT MSVC)
-        set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -static ")
+        set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static -Wl,--whole-archive -Wl,--no-whole-archive -static ")
```

Comment:
Why is this change necessary? What happens if you link against CMS and want to use CMS's threads? Does STP need to link against `pthread`?

If you want this change, please put it in a separate PR.

Reply:
I don't need this change, just noticed that you have extra linkage parameter that not needed at all.

Reply:
Yeah, I'm really not sure about this; I'm just being cautious in making only the changes that you need to get your build working.